docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+
+services:
+  spadesx:
+    volumes: 
+      - server:/server
+    ports:
+      - "32887:32887"
+
+volumes:
+  server: # TODO
```

Comment:
Compose files are useful for apps that require multiple containers (for example, a database or load balancer). SpadesX is a standalone project, so I don't think this is necessary.

Reply:
To run multiple servers you would have to make your own compose file since you need to configure your own volumes and ports. Maybe this would be better suited in a new section in the README (`docker run` example for a single server, and compose for multi server).

Reply:
Yea but giving users the base for docker compose is useful. But agreed. Normal docker file should also be implemented alongside compose. Exactly for this purpose.

Reply:
thanks for the feedback guys. Perhaps we can shelve the compose file for this particular PR and explore it later. What do you think?

Reply:
I've changed my mind and I'm leaning towards keeping the compose file; I've just pushed a commit which refines it compared to the initial copy. Please let me know what you think.

Reply:
yep

Reply:
Mainly write the docker file and then we can explore compose